### DISCUSSION TITLE
Some gamepads not being detected

### DISCUSSION THREAD

**Discussion by shandshellin** [DustinBrett/daedalOS#416] [Feature Requests]
Been having an issue with daedalos detecting my gamepad, tried wired and bt connection but no dice. Works fine in the host os. 8BitDo SN30 pro controller. This did work with an old xbox controller, thoughts?

**Comment by DustinBrett** [comment]
I am using a slightly old and cut down version of [EmulatorJS](https://emulatorjs.org/), perhaps you [try](https://demo.emulatorjs.org/) their emulator and see if it works better.

I haven't done much testing with gamepads and didn't spend much time porting it into daedalOS properly in that way. But if it works for you on the real emulator than perhaps I can fix why it doesn't work in my version.

**Comment by DustinBrett** [comment]
Actually, what are you using the gamepad for, which game/rom/file? I am not clear what the use case is now that I think about it, I assumed it was EmulatorJS as very few other things even support a gamepad on my site.

**Comment by shandshellin** [comment]
Emulatorjs with super smash brothers n64 rom.

**Comment by DustinBrett** [comment]
> Emulatorjs with super smash brothers n64 rom.

Ok good to know. Perhaps you can [try](https://demo.emulatorjs.org/) their emulator and see if it works better? If so I will look into redoing my porting of their library.

**Comment by DustinBrett** [comment]
I will move this to discussion for now as the game controllers are dependent on EmulatorJs, so if it works there I can look at improving the port, but either way it's all about the port as I don't manual do the code currently for any of those emulators.
